docs/2.9.0/docs/query/pqs-user-guide.rst

```diff
@@ -1,466 +1,1741 @@
+Offsets are deeply embedded in the Read API as a way for users to query
+the ledger in a manner that preserves the level of consistency they
+require. Users are able to nominate the offsets they wish to query from,
+or simply query the latest available offset.
 
-.. image:: ./images/offset-sequence.svg
-   :alt: Charts of offsets and transactions for two participants connected by a sync domain
+The following figure shows a pair of participants and their respective
+ledgers. Each participant has its own PQS instance, and you can see that
+it always has the portion of the ledger that it is authorized to see:
 
-You cannot compare offset values across participants. The same ledger change (such as a transaction) for multiple participant nodes is stored at a different offset in each participant node. For example, in the figure, the transaction ABC is at offset #011 in Participant A but at offset #010 in Participant B. Similarly, the same offset value across participant nodes refers to different ledger changes. In the figure, Participant A's offset #011 records “Tx ABC” while Participant B's offset #011 records “Tx DEF”. Comparing offsets across sync domains does not provide a causal ordering of the events because there is no common reference.
+|image4|
 
-Single offset values returned by the Ledger API can be used as-is (for example, to keep track of processed transactions and provide an application restart point in case you need to retry the request).
+We can also see that the offsets (prefix) are common to the Participant
+and PQS, but the Transaction ID's (suffix) are shared throughout.
 
-The Ledger API endpoints that take offset values also allow an offset range which is a portion of the participant's ledger that is relevant for the client to read. An offset range is analogous to a duration or slice of time. For example, a client application can specify the offset range from #010 through #012 to perform a search within Participant B's ledger.
+Offset Management
+-----------------
 
-Although the figure shows integers as an offset value, this may change. The format of offsets should be treated as opaque to the client. No client-side transformation of an offset (such as subtracting or adding one offset to another and using that in a ledger read)  is guaranteed to return a meaningful offset. However, you can always expect that offsets are lexicographically comparable.
+The following functions control the temporal perspective of the ledger,
+and allows you to control how you wish to consider time in your queries.
+Since PQS exposes an eventually-consistent perspective of the ledger,
+you may wish to:
 
-A transaction ID compares to an offset in the following ways:
+-  **Ignore**; simply query the *latest available* state
+-  **Pin**; query the state of the ledger at a specific time in history
```

Comment:
Is this a 'wall clock time' or an 'offset" because "specific time in history" is ambiguous to a new user?

Reply:
You can either pin via time, or offset. I think this sentence refers to a concept, there are many different implementations (record time, effective time, offset..) you can use.